packages/docs/locales/en/en.json

```diff
@@ -244,10 +244,27 @@
         "dropzone": "Enables Drag&Drop",
         "value": "The array with uploaded files"
       }
+    },
+    "VaCheckbox": {
+      "props": {
+        "value": "The value of the checkbox",
+        "arrayValue": "Same as native `value`. It is used with `v-model` of an array type",
+        "label": "Same as native `label`",
+        "leftLabel": "Moves the label on the left of a checkbox",
+        "trueValue": "Overrides the returned value when it's checked",
+        "falseValue": "Overrides the returned value when it's not checked",
+        "indeterminate": "Same as native `indeterminate`",
```

Comment:
Some details would be nice to have. We're not expecting all users to know html spec by heart.